COMPOUND_COMPONENTS.md

```diff
@@ -0,0 +1,466 @@
+# Compound Components
+
+## What is a compound component?
+
+Compound components is a pattern in which components are used together in such a way that describes a higher level of abstraction. A desktop computer is composed of many components that can be serviced and configured individually. Similarly, a compound component allows simultaneous configuration through access to each sub-component. Here are the parts of a compound component:
```

Comment:
What do you think about this for an alternative to the opening line here?
```
> Compound components is a pattern where higher level components are composed using smaller components, and you retain control over how those smaller pieces are wired together.
```

Reply:
I would even consider formatting it in that way, as a quote, so it stands out and apart from the elaborating paragraph.

Reply:
I'm not sure we retain control over how the smaller pieces are wired together (at least not from the component API). It allows configuration of both the overall behavior as well as of individual sub-components by providing direct access to semantic elements.

How about:
```suggestion
> Compound components is a pattern where higher level components are composed using smaller components, and you retain access to all the semantic elements of the higher level component.
```

Reply:
Thanks, that's what I was going for.